### ISSUE TITLE
Is there a guide on how to replace Spring Boot Web Jackson Deserializer with Kotlin Serialization?

### ISSUE THREAD

**Issue by xetra11** [Kotlin/kotlinx.serialization#1490]
I want to use kotlin serialization in my project. However my Spring Boot Web project currently uses Jackson do deserialize i.e. `@RequestBody` objects. I'd like it to use Kotlinx Serializer so I have only one Serialization Library in my project instead of two.

Is there any guide on how to do that around?

**Comment by sandwwraith** [comment]
Not sure we have one. Maybe such a guide should be provided by Spring

**Comment by GeorgePap-719** [comment]
Spring has provided a guide on how to [setup kotlinx.serialization](https://docs.spring.io/spring-framework/reference/languages/kotlin/web.html#kotlin-multiplatform-serialization). 

**tl;dr** Simply add the dependency and plugin and Spring will pick it up by default. Though you cannot configure to use only one serialization library, because kotlinx.serialization serializes only Kotlin classes, and Spring is a Java based framework. Therefore, Spring will use both kotlinx.serialization and Jackson.
